applications/Makefile

```diff
@@ -8,117 +8,76 @@ AWSMFLAGS= --inline-constant-globals --runtime-globals
 # --whole-archive causes the symbols in the listed static archive to be exported from the resulting *.so
 # https://stackoverflow.com/questions/805555/ld-linker-question-the-whole-archive-option
 CFLAGS=-O3 -flto
-LDFLAGS=-shared -fPIC -Wl,--export-dynamic,--whole-archive -L../libsledge/dist/ -lsledge -Wl,--no-whole-archive -lm
```

Comment:
Why remove the shared option?

Reply:
It was down in the rule itself for some reason. Moved back up here.